components/indexComponents/preview.tsx

```diff
@@ -15,8 +16,9 @@ export function Preview() {
         <PrevCompany />
       </div>
       <Link href={"./firmy"}>
```

Comment:
Kropek raczej się nie używa przy podawaniu URL strony. Klamry również nie są wymagane w tym wypadku,
```suggestion
      <Link href="/firmy">
```